employees/migrations/0012_alter_employeehistory_added_date.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1.1 on 2024-03-13 07:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('employees', '0011_employeehistory'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='employeehistory',
+            name='added_date',
+            field=models.DateTimeField(),
+        ),
+    ]
```

Comment:
No need to create multiple migrations for single feature within single PR. Remove both of them & re-generate 1 file again & commit.